sources/packages/backend/apps/api/src/route-controllers/confirmation-of-enrollment/confirmation-of-enrollment.aest.controller.ts

```diff
@@ -0,0 +1,62 @@
+import { Controller, Param, ParseIntPipe, Patch } from "@nestjs/common";
+import {
+  ApiNotFoundResponse,
+  ApiTags,
+  ApiUnprocessableEntityResponse,
+} from "@nestjs/swagger";
+import { AuthorizedParties } from "../../auth/authorized-parties.enum";
+import {
+  AllowAuthorizedParty,
+  Groups,
+  Roles,
+  UserToken,
+} from "../../auth/decorators";
+import { Role } from "../../auth/roles.enum";
+import { UserGroups } from "../../auth/user-groups.enum";
+import { IUserToken } from "../../auth/userToken.interface";
+import { COE_NOT_FOUND_MESSAGE } from "../../constants";
+import { ClientTypeBaseRoute } from "../../types";
+import { COE_WINDOW } from "../../utilities";
+import BaseController from "../BaseController";
+import { ConfirmationOfEnrollmentControllerService } from "./confirmation-of-enrollment.controller.service";
+
+@AllowAuthorizedParty(AuthorizedParties.aest)
+@Groups(UserGroups.AESTUser)
+@Controller("confirmation-of-enrollment")
+@ApiTags(`${ClientTypeBaseRoute.AEST}[Confirmation Of Enrollment - CoE]`)
+export class ConfirmationOfEnrollmentAESTController extends BaseController {
+  constructor(
+    private readonly confirmationOfEnrollmentControllerService: ConfirmationOfEnrollmentControllerService,
+  ) {
+    super();
+  }
+
+  /**
+   * Approve confirmation of enrollment(COE).
+   * An application can have up to two COEs based on the disbursement schedule,
+   * hence the COE approval happens twice for application with more than once disbursement.
+   * Irrespective of number of COEs to be approved, application status is set to complete
+   * on first COE approval.
+   * @param disbursementScheduleId disbursement schedule id of COE.
+   */
+  @ApiNotFoundResponse({ description: COE_NOT_FOUND_MESSAGE })
+  @ApiUnprocessableEntityResponse({
+    description:
+      `Confirmation of enrollment window is greater than ${COE_WINDOW} days ` +
```

Comment:
mm.. I think I heard there would be no 21 day check for AEST users right?  Am I missing something

Reply:
pls review the description